torch/csrc/distributed/rpc/rref_impl.h

```diff
@@ -366,16 +366,17 @@ class TORCH_API OwnerRRef final : public RRef {
   // Has a value or error been set?
   bool hasValue() const;
   // Gets a future that is satisfied when the value or error is set.
-  std::shared_ptr<FutureMessage> getFuture();
+  std::shared_ptr<FutureIValue> getFuture();
 
  private:
   friend class RRefContext;
 
+  // TODO: consolidate value_ into future_
```

Comment:
I tried to do this in https://github.com/pytorch/pytorch/pull/32608

Reply:
I see, thanks for the reminder. Let me replace the TODO with a pointer to that abandoned PR then.